packages/typography/src/elements/Ellipsis.tsx

```diff
@@ -10,11 +10,9 @@ import PropTypes from 'prop-types';
 import { StyledEllipsis } from '../styled';
 
 interface IEllipsisProps extends HTMLAttributes<HTMLSpanElement> {
-  /**
-   * Optional override for the auto-generated `title` attribute
-   */
+  /** Overrides the auto-generated `title` attribute */
   title?: string;
-  /** Any valid element for the styled component */
+  /** Updates the element tag */
```

Comment:
~tag~ -> "HTML tag"? Or is "element's HTML tag" too much? ditto for similar props below...

Reply:
I think "element's HTML tag" works!